### PR TITLE
powerplatform-vscode Loc checking for WO71412

### PR DESCRIPTION
**Pull request by v-hifuku-msft**
Localized new strings up to early day Monday, September 12.

### CODE REVIEW

File: i18n/fra/package.i18n.json [hunk 1]
```diff
@@ -36,5 +36,13 @@
...
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.title": "Fonctionnalités avancées",
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.description": "Visual Studio Code pour le Web permet de modifier et de publier des pages Web sur votre site Web.\n \n Si vous souhaitez utiliser une interface de ligne de commande et des fonctionnalités plus avancées, installez l’extension Power Platform pour VS Code, qui est disponible dans le Marketplace VS Code pour le bureau. \n \n [En savoir plus](command:powerplatform-walkthrough.advancedCapabilities-learn-more) sur la différence entre Visual Studio Code pour le bureau et le web.\n \n[Commencez le codage] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)"
```
Comment (tehcrashxor) [inline i18n/fra/package.i18n.json:47]: > `[Commencez le codage] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)`

No spaces permitted between the `]` and `(` in the markdown link format.

Comment (tehcrashxor) [inline i18n/fra/package.i18n.json:47]: > `[Commencez le codage] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)`

No spaces permitted between the `]` and `(` in the markdown link format.

File: i18n/ita/package.i18n.json [hunk 2]
```diff
@@ -36,5 +36,13 @@
...
+	"microsoft-powerapps-portals.walkthrough.overview.title": "Panoramica",
+	"microsoft-powerapps-portals.walkthrough.overview.description": "L'estensione Power Pages abilita un file system personalizzato che viene sincronizzato con la struttura di progetto dello studio di Power Pages. \n \n L'anteprima è collegata a Power Pages e consente di visualizzare le modifiche.\n[Altre informazioni](command:powerplatform-walkthrough.overview-learn-more)",
+	"microsoft-powerapps-portals.walkthrough.fileSystem.title": "File system",
+	"microsoft-powerapps-portals.walkthrough.fileSystem.description": "I file vengono archiviati in cartelle annidate sotto il nome del sito. \n \n I singoli file di pagine sono disponibili nella cartella delle pagine Web sotto il nome del sito. \n \n Tutte le pagine sono organizzate in file HTML, CSS e JS all'interno della cartella con il rispettivo nome di pagina. \n \n Per altre informazioni sul file system Power Pages in VS Code per il Web, visita la nostra [documentazione](command:powerplatform-walkthrough.fileSystem-documentation). \n [Apri cartella del sito] (command:powerplatform-walkthrough.fileSystem-open-folder)",
```
Comment (tehcrashxor) [inline i18n/ita/package.i18n.json:45]: No spaces permitted between the `]` and `(` in the markdown link format.   
(2 in this file)

File: i18n/ptb/package.i18n.json [hunk 3]
```diff
@@ -36,5 +36,13 @@
 	"pacCLI.envAndSolutionsPanel.copyVersionNumber.title": "Copiar Número da Versão",
 	"microsoft-powerapps-portals.preview-show.title": "Portal do PowerApps -> Mostrar versão preliminar",
 	"microsoft-powerapps-portals.webExtension.init.title": "Inicializar Extensão da Web",
-	"microsoft-powerapps-portals.webExtension.limitation": "Não há suporte para PAC CLI no VS Code para a Web"
+	"microsoft-powerapps-portals.webExtension.limitation": "Não há suporte para PAC CLI no VS Code para a Web",
+	"microsoft-powerapps-portals.walkthrough.title": "Editar código do Power Pages",
+	"microsoft-powerapps-portals.walkthrough.description": "Descubra como você pode criar sites completos e modernos, diretamente no VS Code",
+	"microsoft-powerapps-portals.walkthrough.overview.title": "Visão Geral",
+	"microsoft-powerapps-portals.walkthrough.overview.description": "A extensão do Power Pages permite um sistema de arquivos personalizado que é sincronizado com a estrutura do projeto do estúdio do Power Pages.\n \n A visualização está vinculada ao Power Pages e permite que você veja suas alterações.\n[Saiba Mais] (command:powerplatform-walkthrough.overview-learn-more)",
```
Comment (tehcrashxor) [inline i18n/ptb/package.i18n.json:43]: No spaces permitted between the `]` and `(` in the markdown link format.   
(3 in this file)

File: i18n/rus/package.i18n.json [hunk 4]
```diff
@@ -36,5 +36,13 @@
...
+	"microsoft-powerapps-portals.walkthrough.description": "Узнайте, как создавать работающие современные веб-сайты в VS Code",
+	"microsoft-powerapps-portals.walkthrough.overview.title": "Обзор",
+	"microsoft-powerapps-portals.walkthrough.overview.description": "Расширение Power Pages позволяет работать с файловой системой, которая синхронизируется со структурой проекта в студии Power Pages.\n \n Предварительный просмотр страниц доступен по ссылкам Power Pages.\n[Подробнее](command:powerplatform-walkthrough.overview-learn-more)",
+	"microsoft-powerapps-portals.walkthrough.fileSystem.title": "Файловая система",
+	"microsoft-powerapps-portals.walkthrough.fileSystem.description": "Файлы хранятся во вложенных папках в папке с именем сайта. \n \n Файлы отдельных страниц можно найти в папке веб-страниц. \n \n Все страницы состоят из файлов HTML, CSS и JS, находящихся в папке с именем соответствующей страницы. \n \n Подробнее о файловой системе Power Pages в веб-версии VS Code читайте в [документации](command:powerplatform-walkthrough.fileSystem-documentation). \n[Открыть папку сайта] (command:powerplatform-walkthrough.fileSystem-open-folder)",
```
Comment (tehcrashxor) [inline i18n/rus/package.i18n.json:45]: No spaces permitted between the `]` and `(` in the markdown link format.   
(2 in this file)

File: loc/translations-import/vscode-powerplatform.fr.xlf [hunk 5]
```diff
@@ -184,6 +201,13 @@ L’authentification interactive n’est pas disponible pour les scénarios dist
  [Learn More](command:powerplatform-walkthrough.advancedCapabilities-learn-more) about the difference between Visual Studio Code for desktop and web.
  
 [Start coding](command:powerplatform-walkthrough.advancedCapabilities-start-coding)</source>
+        <target state="translated">Visual Studio Code pour le Web permet de modifier et de publier des pages Web sur votre site Web.
+ 
+ Si vous souhaitez utiliser une interface de ligne de commande et des fonctionnalités plus avancées, installez l’extension Power Platform pour VS Code, qui est disponible dans le Marketplace VS Code pour le bureau. 
+ 
+ [En savoir plus](command:powerplatform-walkthrough.advancedCapabilities-learn-more) sur la différence entre Visual Studio Code pour le bureau et le web.
+ 
+[Commencez le codage] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)</target>
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.fr.xlf:210]: No spaces permitted between the `]` and `(` in the markdown link format.

File: loc/translations-import/vscode-powerplatform.it.xlf [hunk 6]
```diff
@@ -172,9 +180,18 @@ L'autenticazione interattiva non è disponibile per scenari remoti. I profili di
...
  To learn more about the Power Pages file system in VS Code for Web, visit our [documentation](command:powerplatform-walkthrough.fileSystem-documentation). 
 [Open site folder](command:powerplatform-walkthrough.fileSystem-open-folder)</source>
+        <target state="translated">I file vengono archiviati in cartelle annidate sotto il nome del sito. 
+ 
+ I singoli file di pagine sono disponibili nella cartella delle pagine Web sotto il nome del sito. 
+ 
+ Tutte le pagine sono organizzate in file HTML, CSS e JS all'interno della cartella con il rispettivo nome di pagina. 
+ 
+ Per altre informazioni sul file system Power Pages in VS Code per il Web, visita la nostra [documentazione](command:powerplatform-walkthrough.fileSystem-documentation). 
+ [Apri cartella del sito] (command:powerplatform-walkthrough.fileSystem-open-folder)</target>
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.it.xlf:190]: No spaces permitted between the `]` and `(` in the markdown link format.

File: loc/translations-import/vscode-powerplatform.it.xlf [hunk 7]
```diff
@@ -184,6 +201,13 @@ L'autenticazione interattiva non è disponibile per scenari remoti. I profili di
  [Learn More](command:powerplatform-walkthrough.advancedCapabilities-learn-more) about the difference between Visual Studio Code for desktop and web.
  
 [Start coding](command:powerplatform-walkthrough.advancedCapabilities-start-coding)</source>
+        <target state="translated">Visual Studio Code per il Web consente la modifica e la pubblicazione di pagine Web sul sito Web.
+ 
+ Per un'interfaccia della riga di comando e funzionalità più avanzate, installa l'estensione Power Platform per VS Code, disponibile in VS Code Marketplace per desktop. 
+ 
+ [Altre informazioni](command:powerplatform-walkthrough.advancedCapabilities-learn-more) sulla differenza tra Visual Studio sulla Code per desktop e per il Web.
+ 
+[Codifica iniziale] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)</target>
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.it.xlf:210]: No spaces permitted between the `]` and `(` in the markdown link format.

File: loc/translations-import/vscode-powerplatform.pt-BR.xlf [hunk 8]
```diff
@@ -147,21 +147,29 @@ A Autenticação Interativa não está disponível para cenários remotos; os pe
...
       </trans-unit>
       <trans-unit id="microsoft-powerapps-portals.walkthrough.overview.description">
         <source xml:lang="en">Power Pages extension enables a customized file system which is synced with the project structure of the Power Pages studio.
  
  The preview is linked to Power Pages and enables you to view your changes.
 [Learn More](command:powerplatform-walkthrough.overview-learn-more)</source>
+        <target state="translated">A extensão do Power Pages permite um sistema de arquivos personalizado que é sincronizado com a estrutura do projeto do estúdio do Power Pages.
+ 
+ A visualização está vinculada ao Power Pages e permite que você veja suas alterações.
+[Saiba Mais] (command:powerplatform-walkthrough.overview-learn-more)</target>
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.pt-BR.xlf:168]: No spaces permitted between the `]` and `(` in the markdown link format.

File: loc/translations-import/vscode-powerplatform.pt-BR.xlf [hunk 9]
```diff
@@ -172,9 +180,18 @@ A Autenticação Interativa não está disponível para cenários remotos; os pe
...
  To learn more about the Power Pages file system in VS Code for Web, visit our [documentation](command:powerplatform-walkthrough.fileSystem-documentation). 
 [Open site folder](command:powerplatform-walkthrough.fileSystem-open-folder)</source>
+        <target state="translated">Seus arquivos são armazenados em pastas aninhadas no nome do site. 
+ 
+ Os arquivos de página individuais podem ser encontrados na pasta páginas da Web com o nome do site. 
+ 
+ Todas as suas páginas são organizadas em arquivos HTML, CSS e JS dentro da pasta com o respectivo nome de página. 
+ 
+ Para saber mais sobre o sistema de arquivos do Power Pages no VS Code para Web, acesse nossa [documentação](command:powerplatform-walkthrough.fileSystem-documentation). 
+[Abrir pasta do site] (command:powerplatform-walkthrough.fileSystem-open-folder)</target>
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.pt-BR.xlf:190]: No spaces permitted between the `]` and `(` in the markdown link format.

File: loc/translations-import/vscode-powerplatform.pt-BR.xlf [hunk 10]
```diff
@@ -184,6 +201,13 @@ A Autenticação Interativa não está disponível para cenários remotos; os pe
  [Learn More](command:powerplatform-walkthrough.advancedCapabilities-learn-more) about the difference between Visual Studio Code for desktop and web.
  
 [Start coding](command:powerplatform-walkthrough.advancedCapabilities-start-coding)</source>
+        <target state="translated">Visual Studio Code para Web permite a edição e publicação de páginas da Web em seu site.
+ 
+ Para obter uma interface de linha de comando e recursos mais avançados, instale a Extensão do Power Platform para o VS Code, disponível no Marketplace do VS Code para desktop. 
+ 
+ [Saiba Mais](command:powerplatform-walkthrough.advancedCapabilities-learn-more) sobre a diferença entre o Visual Studio Code para desktop e para Web.
+ 
+[Iniciar a codificação] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)</target>
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.pt-BR.xlf:210]: No spaces permitted between the `]` and `(` in the markdown link format.

File: loc/translations-import/vscode-powerplatform.ru.xlf [hunk 11]
```diff
@@ -172,9 +180,18 @@ The third line should be '[TRANSLATION HERE](command:pacCLI.pacAuthHelp)', keepi
...
  To learn more about the Power Pages file system in VS Code for Web, visit our [documentation](command:powerplatform-walkthrough.fileSystem-documentation). 
 [Open site folder](command:powerplatform-walkthrough.fileSystem-open-folder)</source>
+        <target state="translated">Файлы хранятся во вложенных папках в папке с именем сайта. 
+ 
+ Файлы отдельных страниц можно найти в папке веб-страниц. 
+ 
+ Все страницы состоят из файлов HTML, CSS и JS, находящихся в папке с именем соответствующей страницы. 
+ 
+ Подробнее о файловой системе Power Pages в веб-версии VS Code читайте в [документации](command:powerplatform-walkthrough.fileSystem-documentation). 
+[Открыть папку сайта] (command:powerplatform-walkthrough.fileSystem-open-folder)</target>
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.ru.xlf:190]: No spaces permitted between the `]` and `(` in the markdown link format.

File: loc/translations-import/vscode-powerplatform.ru.xlf [hunk 12]
```diff
@@ -184,6 +201,13 @@ The third line should be '[TRANSLATION HERE](command:pacCLI.pacAuthHelp)', keepi
  [Learn More](command:powerplatform-walkthrough.advancedCapabilities-learn-more) about the difference between Visual Studio Code for desktop and web.
  
 [Start coding](command:powerplatform-walkthrough.advancedCapabilities-start-coding)</source>
+        <target state="translated">Веб-версия Visual Studio Code позволяет редактировать и публиковать страницы веб-сайта.
+ 
+ Для работы с интерфейсом командной строки и использования расширенных функций установите расширение Power Platform для VS Code из магазина классической версии VS Code. 
+ 
+ [Подробнее](command:powerplatform-walkthrough.advancedCapabilities-learn-more) о различиях между классической и веб-версией Visual Studio Code.
+ 
+[Писать код] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)</target>
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.ru.xlf:210]: No spaces permitted between the `]` and `(` in the markdown link format.

File: package.nls.fr.json [hunk 13]
```diff
@@ -32,12 +32,12 @@
...
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.title": "Fonctionnalités avancées",
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.description": "Visual Studio Code pour le Web permet de modifier et de publier des pages Web sur votre site Web.\n \n Si vous souhaitez utiliser une interface de ligne de commande et des fonctionnalités plus avancées, installez l’extension Power Platform pour VS Code, qui est disponible dans le Marketplace VS Code pour le bureau. \n \n [En savoir plus](command:powerplatform-walkthrough.advancedCapabilities-learn-more) sur la différence entre Visual Studio Code pour le bureau et le web.\n \n[Commencez le codage] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)"
```
Comment (tehcrashxor) [inline package.nls.fr.json:42]: No spaces permitted between the `]` and `(` in the markdown link format.

File: package.nls.it.json [hunk 14]
```diff
@@ -32,12 +32,12 @@
...
+	"microsoft-powerapps-portals.walkthrough.fileSystem.description": "I file vengono archiviati in cartelle annidate sotto il nome del sito. \n \n I singoli file di pagine sono disponibili nella cartella delle pagine Web sotto il nome del sito. \n \n Tutte le pagine sono organizzate in file HTML, CSS e JS all'interno della cartella con il rispettivo nome di pagina. \n \n Per altre informazioni sul file system Power Pages in VS Code per il Web, visita la nostra [documentazione](command:powerplatform-walkthrough.fileSystem-documentation). \n [Apri cartella del sito] (command:powerplatform-walkthrough.fileSystem-open-folder)",
```
Comment (tehcrashxor) [inline package.nls.it.json:40]: No spaces permitted between the `]` and `(` in the markdown link format.

File: package.nls.it.json [hunk 15]
```diff
@@ -32,12 +32,12 @@
...
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.title": "Funzionalità avanzate",
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.description": "Visual Studio Code per il Web consente la modifica e la pubblicazione di pagine Web sul sito Web.\n \n Per un'interfaccia della riga di comando e funzionalità più avanzate, installa l'estensione Power Platform per VS Code, disponibile in VS Code Marketplace per desktop. \n \n [Altre informazioni](command:powerplatform-walkthrough.advancedCapabilities-learn-more) sulla differenza tra Visual Studio sulla Code per desktop e per il Web.\n \n[Codifica iniziale] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)"
```
Comment (tehcrashxor) [inline package.nls.it.json:42]: No spaces permitted between the `]` and `(` in the markdown link format.

File: package.nls.pt-BR.json [hunk 16]
```diff
@@ -32,12 +32,12 @@
...
+	"microsoft-powerapps-portals.walkthrough.title": "Editar código do Power Pages",
+	"microsoft-powerapps-portals.walkthrough.description": "Descubra como você pode criar sites completos e modernos, diretamente no VS Code",
+	"microsoft-powerapps-portals.walkthrough.overview.title": "Visão Geral",
+	"microsoft-powerapps-portals.walkthrough.overview.description": "A extensão do Power Pages permite um sistema de arquivos personalizado que é sincronizado com a estrutura do projeto do estúdio do Power Pages.\n \n A visualização está vinculada ao Power Pages e permite que você veja suas alterações.\n[Saiba Mais] (command:powerplatform-walkthrough.overview-learn-more)",
```
Comment (tehcrashxor) [inline package.nls.pt-BR.json:38]: No spaces permitted between the `]` and `(` in the markdown link format.

File: package.nls.pt-BR.json [hunk 17]
```diff
@@ -32,12 +32,12 @@
...
+	"microsoft-powerapps-portals.walkthrough.fileSystem.title": "Sistema de arquivos",
+	"microsoft-powerapps-portals.walkthrough.fileSystem.description": "Seus arquivos são armazenados em pastas aninhadas no nome do site. \n \n Os arquivos de página individuais podem ser encontrados na pasta páginas da Web com o nome do site. \n \n Todas as suas páginas são organizadas em arquivos HTML, CSS e JS dentro da pasta com o respectivo nome de página. \n \n Para saber mais sobre o sistema de arquivos do Power Pages no VS Code para Web, acesse nossa [documentação](command:powerplatform-walkthrough.fileSystem-documentation). \n[Abrir pasta do site] (command:powerplatform-walkthrough.fileSystem-open-folder)",
```
Comment (tehcrashxor) [inline package.nls.pt-BR.json:40]: No spaces permitted between the `]` and `(` in the markdown link format.

File: package.nls.pt-BR.json [hunk 18]
```diff
@@ -32,12 +32,12 @@
...
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.title": "Recursos avançados",
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.description": "Visual Studio Code para Web permite a edição e publicação de páginas da Web em seu site.\n \n Para obter uma interface de linha de comando e recursos mais avançados, instale a Extensão do Power Platform para o VS Code, disponível no Marketplace do VS Code para desktop. \n \n [Saiba Mais](command:powerplatform-walkthrough.advancedCapabilities-learn-more) sobre a diferença entre o Visual Studio Code para desktop e para Web.\n \n[Iniciar a codificação] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)"
```
Comment (tehcrashxor) [inline package.nls.pt-BR.json:42]: No spaces permitted between the `]` and `(` in the markdown link format.

File: package.nls.ru.json [hunk 19]
```diff
@@ -32,12 +32,12 @@
...
+	"microsoft-powerapps-portals.walkthrough.description": "Узнайте, как создавать работающие современные веб-сайты в VS Code",
+	"microsoft-powerapps-portals.walkthrough.overview.title": "Обзор",
+	"microsoft-powerapps-portals.walkthrough.overview.description": "Расширение Power Pages позволяет работать с файловой системой, которая синхронизируется со структурой проекта в студии Power Pages.\n \n Предварительный просмотр страниц доступен по ссылкам Power Pages.\n[Подробнее](command:powerplatform-walkthrough.overview-learn-more)",
+	"microsoft-powerapps-portals.walkthrough.fileSystem.title": "Файловая система",
+	"microsoft-powerapps-portals.walkthrough.fileSystem.description": "Файлы хранятся во вложенных папках в папке с именем сайта. \n \n Файлы отдельных страниц можно найти в папке веб-страниц. \n \n Все страницы состоят из файлов HTML, CSS и JS, находящихся в папке с именем соответствующей страницы. \n \n Подробнее о файловой системе Power Pages в веб-версии VS Code читайте в [документации](command:powerplatform-walkthrough.fileSystem-documentation). \n[Открыть папку сайта] (command:powerplatform-walkthrough.fileSystem-open-folder)",
```
Comment (tehcrashxor) [inline package.nls.ru.json:40]: No spaces permitted between the `]` and `(` in the markdown link format.

File: package.nls.ru.json [hunk 20]
```diff
@@ -32,12 +32,12 @@
...
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.title": "Расширенные возможности",
+	"microsoft-powerapps-portals.walkthrough.advancedCapabilities.description": "Веб-версия Visual Studio Code позволяет редактировать и публиковать страницы веб-сайта.\n \n Для работы с интерфейсом командной строки и использования расширенных функций установите расширение Power Platform для VS Code из магазина классической версии VS Code. \n \n [Подробнее](command:powerplatform-walkthrough.advancedCapabilities-learn-more) о различиях между классической и веб-версией Visual Studio Code.\n \n[Писать код] (command:powerplatform-walkthrough.advancedCapabilities-start-coding)"
```
Comment (tehcrashxor) [inline package.nls.ru.json:42]: No spaces permitted between the `]` and `(` in the markdown link format.

**Review by tehcrashxor — CHANGES_REQUESTED** [review]

**Comment by v-hifuku-msft** [comment]
@Toshio ***@***.***>,

Please let me know how you want to fix this.

Hiro

**Comment by v-hifuku-msft** [comment]
Abandoned this PR. New PR will be raised with fixes.

**Pull request closed without merging**